fbw-a380x/src/systems/instruments/src/ANC/instrument.tsx

```diff
@@ -0,0 +1,141 @@
+import { EventBus, FSComponent, HEventPublisher } from '@microsoft/msfs-sdk';
+
+var flightphase = {
+  "WelcomeOnboard": 0,
+  "SafetyBriefing": 0,
+  "CrewSeatsTakeoff": 0,
+  "SeatbeltSignOff": 0,
+  "StartingDecent": 0,
+  "CrewPrepareLanding": 0,
+  "ArrivalDeboarding": 0
+};
```

Comment:
Are these exclusive? Why not an enum and a state machine?

Reply:
these were created exclusive to make sure that the announcement would also play when users would not fill out the flightplan. But I can also revert it and make it based on the enums

Reply:
`var` should also never be used.